### ISSUE TITLE
Help with replicating table 3

### ISSUE THREAD

**Issue by jiahai-feng** [amounts-tidings/fact_learning#1]
Hi, thanks for sharing your results and code. The results look very interesting!

I'm trying to replicate Table 3, and I'm wondering if you could help me out.

Based on your readme file, it seems like I should first train using

```
python train.py \
    --config-name finetune_full.yaml \
    model=meta-llama/Meta-Llama-3-8B \
    subset=Corpus_narrative \
    save=save/llama3_8b_narrative
```

And then evaluate using
```
python lm-evaluation-harness/main.py \
    --model hf-causal-experimental \
    --model_args pretrained=save/llama3_8b_narrative,use_accelerate=True \
    --no_cache \
    --tasks Eval_QA \
    --num_fewshot 5 \
    --batch_size 4
```
Doing this gives me 67.5 accuracy, which is quite different from the reported 100. I tried finetuning the model with a different seed, and got 72.5. Do you think this is because the hyperparameters in `finetune_full.yaml` are not correct?

In addition, it doesn't look like `train.py` contains code for the baseline where you only finetune the first 10 layers, or code for implementing active forgetting with full finetuning. Could you give some pointers to how I can replicate these particular experiments?
